src/darwin/Framework/CHIPTests/TestHelpers/MTRTestDeclarations.h

```diff
@@ -59,10 +59,31 @@ NS_ASSUME_NONNULL_BEGIN
 @end
 
 @interface MTRDevice (TestDebug)
+typedef NS_ENUM(NSUInteger, MTRInternalDeviceState) {
```

Comment:
Why is this here?  Seems like we should just move the "internal device state" to a header that both MTRDevice.mm and the tests include instead of duplicating the enum and having problems when our duplication gets out of sync.

Reply:
done.